data/sql/updates/pending_db_world/rev_1639254662548269400.sql

```diff
@@ -0,0 +1,132 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1639254662548269400');
+
+UPDATE `game_event` SET `holiday`=375, `length`=4320, `holidayStage`=1 WHERE `eventEntry`=71;
+
+DELETE FROM `game_event` WHERE `eventEntry`=77;
+INSERT INTO `game_event` VALUES
+(77,'2016-11-06 02:01:00','2030-12-31 07:00:00',131040,4320,376,1,'Darkmoon Faire Building (Terokkar Forest)',0,2);
+
+DELETE FROM `creature` WHERE `guid` IN (2000000,2000001,2000002,2000003,2000004,2000005,2000006,2000007);
+INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`) VALUES
+(2000000, 14849, 530, 1, 1, 0, 1, -2250.29, 5245.24, -9.85045, 5.03122, 300, 0, 0, 699, 0, 0),
```

Comment:
these GUID's are high af. Please make them lower.

Reply:
I don't want to use lower guids, because these might be used by other TC cherry picks.

Reply:
Use the guids TC are using?

Reply:
Already used in our DB. That's why I want to prevent from conflicts.

Reply:
Also it doesnt matter if other have higher GUIDs, we shouldn't ever need to reach these guid's

Reply:
I used guids that I'm sure TC will never use (because the same reason as you do), so it is safe to use them to prevent DB guids conficts. It's a rare situation. The guids used by TC are already in use in our DB - now we have confict. If I use something different (lower) guids, in future again might be some conflicts. And recently we are cherry-picking commits from TC very frequently.